### ISSUE TITLE
Software compatibility

### ISSUE THREAD

**Issue by alpineyahoo** [ohmplatform/freedom-gpt-electron-app#1]
Hi. Does the application(arm64) work on **Intel** Mac?

**Comment by lewisje** [comment]
No: Apple has not made a forward-compatibility solution to allow ARM code to run on Intel CPUs, even though it does have a backward-compatibility solution to allow Intel x86_64 code to run on Apple silicon.

**Comment by alpineyahoo** [comment]
@lewisje 
Are you a contributor to FreedomGPT project?
It would be great if you make one for **Intel** Mac.

**Comment by lewisje** [comment]
No, sorry.
